map/views.py

```diff
@@ -30,6 +32,38 @@ def handle_invalid_usage(error):
     return response
 
 
+def get_cascaders() -> List[str]:
+    return list(flask_redis.smembers("cascaders.users"))
+
+
+def find_cascader(cascaders: List[str], hash: str) -> Optional[str]:
+    for uun in cascaders:
+        if check_uun_hash(uun, hash):
+            return uun
+    return None
+
+
+def get_cascader_elsewhere_count(cascaders: List[str], notRoom: str) -> int:
+    if not current_user.is_authenticated:
+        return 7
```

Comment:
Why 7?

Reply:
Random number, for demo

Reply:
Hmm, ok. Better to not have too many magic numbers in the source code. Maybe flag this up to the user somehow?

Reply:
Fixed — this didn't work anyway.